counter/Makefile

```diff
@@ -0,0 +1,29 @@
+PROJ=counter
```

Comment:
Do you need to include this entire project as well?

Reply:
Nope.  I have no idea how that got in there.